packages/block-library/src/navigation/edit/index.js

```diff
@@ -648,56 +673,40 @@ function Navigation( {
 							) }
 					</InspectorControls>
 				) }
-				<nav { ...blockProps }>
-					{ isPlaceholderShown && (
-						<PlaceholderComponent
-							currentMenuId={ ref }
-							onFinish={ ( post ) => {
-								setIsPlaceholderShown( false );
-								if ( post ) {
-									setRef( post.id );
-								}
-								selectBlock( clientId );
-							} }
-							canSwitchNavigationMenu={ canSwitchNavigationMenu }
-							hasResolvedNavigationMenus={
-								hasResolvedNavigationMenus
-							}
-							clientId={ clientId }
-							canUserCreateNavigationMenu={
-								canUserCreateNavigationMenu
-							}
-						/>
-					) }
-					{ ! hasResolvedCanUserCreateNavigationMenu ||
-						( ! isEntityAvailable && ! isPlaceholderShown && (
-							<PlaceholderPreview isLoading />
-						) ) }
-					{ ! isPlaceholderShown && (
-						<ResponsiveWrapper
-							id={ clientId }
-							onToggle={ setResponsiveMenuVisibility }
-							label={ __( 'Menu' ) }
-							hasIcon={ hasIcon }
-							isOpen={ isResponsiveMenuOpen }
-							isResponsive={ isResponsive }
-							isHiddenByDefault={ 'always' === overlayMenu }
-							classNames={ overlayClassnames }
-							styles={ overlayStyles }
-						>
-							{ isEntityAvailable && (
-								<NavigationInnerBlocks
-									isVisible={ ! isPlaceholderShown }
-									clientId={ clientId }
-									hasCustomPlaceholder={
-										!! CustomPlaceholder
-									}
-									orientation={ orientation }
-								/>
-							) }
-						</ResponsiveWrapper>
-					) }
-				</nav>
+
+				{ isLoading && (
+					<TagName { ...blockProps }>
+						<Spinner className="wp-block-navigation__loading-indicator" />
+					</TagName>
+				) }
+
+				{ ! isLoading && ! isPlaceholder && (
```

Comment:
Nit: isPlaceholder is certain to be false at this point.

Reply:
✅